lib/Onyx.js

```diff
@@ -918,11 +946,12 @@ function multiSet(data) {
  * Merges an array of changes with an existing value
  *
  * @private
- * @param {Array<*>} changes Array of changes that should be applied to the existing value
  * @param {*} existingValue
+ * @param {Array<*>} changes Array of changes that should be applied to the existing value
+ * @param {bool} omitNull Whether null top-level null values should be omitted
```

Comment:
The `omitNull` here needs to be removed.

Reply:
right, sorry for that 🥴